src/realm/transaction.cpp

```diff
@@ -104,27 +104,29 @@ void generate_properties_for_obj(Replication& repl, const Obj& obj, const ColInf
 
 namespace realm {
 
-std::map<DB::TransactStage, const char*> log_messages = {
-    {DB::TransactStage::transact_Frozen, "Start frozen: %1"},
-    {DB::TransactStage::transact_Writing, "Start write: %1"},
-    {DB::TransactStage::transact_Reading, "Start read: %1"},
+std::map<DB::TransactStage, const char*> log_stage = {
+    {DB::TransactStage::transact_Frozen, "frozen"},
+    {DB::TransactStage::transact_Writing, "write"},
+    {DB::TransactStage::transact_Reading, "read"},
 };
 
 Transaction::Transaction(DBRef _db, SlabAlloc* alloc, DB::ReadLockInfo& rli, DB::TransactStage stage)
     : Group(alloc)
     , db(_db)
     , m_read_lock(rli)
+    , m_log_id((size_t(this) >> 4) & 0xffff)
```

Comment:
can we avoid having this magic formula in two places in the code?

Reply:
Better now?